src/SFA.DAS.ApplyService.InternalApi.UnitTests/Services/Assessor/AssessorSectorDetailsServiceTests.cs

```diff
@@ -127,7 +127,7 @@ public async Task Get_sector_details_for_page4_answers()
             var organisationDetails = "other organisations";
         
             _extractAnswerValueService
-                .Setup(a => a.ExtractAnswerValueFromQuestionId(It.IsAny<List<AssessorAnswer>>(), _sectorQuestionIds.FullName))
+                .Setup(a => a.ExtractAnswerValueFromQuestionId(It.IsAny<List<AssessorAnswer>>(), _sectorQuestionIds.FirstName))
                 .Returns(fullName);
```

Comment:
first name. Also do you need last name here?

Reply:
yes